node/forest_libp2p/src/service.rs

```diff
@@ -402,9 +406,10 @@ async fn handle_network_message<P: StoreParams>(
 
 #[allow(clippy::too_many_arguments)]
 async fn handle_forest_behaviour_event<DB, P: StoreParams>(
-    bh_mut: &mut ForestBehaviour<P>,
+    st_mut: &mut Swarm<ForestBehaviour<P>>,
```

Comment:
do we need those postfixes? For example just `swarm`, it's short enough.

Reply:
Fixed. That's a better name